### PR TITLE
[4.12] OCPBUGS-6789: Pass featureset to kas

### PR DESCRIPTION
**Pull request by stlaz**
backport of https://github.com/openshift/installer/pull/6579

/cc @deads2k @ibihim 
/hold

requires https://github.com/openshift/cluster-kube-apiserver-operator/pull/1439

### CODE REVIEW

**Comment by deads2k** [comment]
/approve
/assign @ibihim 

for lgtm

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 1]
```diff
@@ -174,6 +174,11 @@ then
 
 	rm --recursive --force kube-apiserver-bootstrap
 
+  ADDITIONAL_FLAGS=()
```
Comment (ibihim) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:177]: nit: indent seems to be off

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 2]
```diff
@@ -205,6 +211,11 @@ then
 
 	rm --recursive --force kube-controller-manager-bootstrap
 
+  ADDITIONAL_FLAGS=()
```
Comment (ibihim) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:214]: indent seems off here too.

File: data/data/bootstrap/files/usr/local/bin/bootkube.sh.template [hunk 3]
```diff
@@ -216,7 +227,8 @@ then
 		--asset-output-dir=/assets/kube-controller-manager-bootstrap \
 		--config-output-file=/assets/kube-controller-manager-bootstrap/config \
 		--cpc-config-output-file=/assets/kube-controller-manager-bootstrap/cpc-config \
-		--cluster-config-file=/assets/manifests/cluster-network-02-config.yml
+		--cluster-config-file=/assets/manifests/cluster-network-02-config.yml \
+    "${ADDITIONAL_FLAGS[@]}"
```
Comment (ibihim) [inline data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:231]: indent seems to be off here too

Comment (stlaz) [reply data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:231]: This is a backport, off indent is off here, it's also off in master. I wouldn't fix it.

Comment (ibihim) [reply data/data/bootstrap/files/usr/local/bin/bootkube.sh.template:231]: hm, ok.

**Review by ibihim — APPROVED** [review]

**Comment by stlaz** [comment]
@openshift/installer can we please have the tags necessary to merge this PR?

**Pull request closed without merging**